components/vni.js

```diff
@@ -0,0 +1,63 @@
+// Vni.js
+
+var _ = require('underscore');
+var Lm = require('./Lm');
+var inputs = require('./input-state');
+
+// Hash of vnis
+vnis = []; 
```

Comment:
What is the scope of vnis? I think it should be specific to the nodeInstance and not be a global variable.